src/Resources/V2/Paypage.php

```diff
@@ -0,0 +1,316 @@
+<?php
+
+namespace UnzerSDK\Resources\V2;
+
+use stdClass;
+use UnzerSDK\Adapter\HttpAdapterInterface;
+use UnzerSDK\Apis\PaypageAPIConfig;
+use UnzerSDK\Constants\TransactionTypes;
+use UnzerSDK\Resources\AbstractUnzerResource;
+use UnzerSDK\Resources\EmbeddedResources\Paypage\Payment;
+use UnzerSDK\Resources\EmbeddedResources\Paypage\PaymentMethodsConfigs;
+use UnzerSDK\Resources\EmbeddedResources\Paypage\Resources;
+use UnzerSDK\Resources\EmbeddedResources\Paypage\Style;
+use UnzerSDK\Resources\EmbeddedResources\Paypage\Urls;
+use UnzerSDK\Resources\EmbeddedResources\RiskData;
+
+class Paypage extends AbstractUnzerResource
+{
+    public const URI = '/merchant/paypage';
+
+    /** @var string $mode "charge" or "authorize" */
+    protected string $mode;
+    protected float $amount;
+    protected string $currency;
+
+    protected ?string $type = null;
```

Comment:
checkoutType as well e.g. full, payment_only, no_shipping

Reply:
- Add checkout type to paypage.
- Remove checkout type from method config.